src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 40px 0 50px;
+}
+
+.logo-img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 10px;
+  white-space: nowrap;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to CSS variables and move them to :root selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)